BenchmarkTests/BenchmarkTests.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,167 @@
+{
```

Comment:
There are quite a few dependencies, are they all required for the Benchmark project?

Reply:
They are transitive dependencies of otel-swift, I have removed the `Package.resolved`